File: app/src/main/java/com/printf/shuttle_tracker/View/Splash/SplashActivity.java
package com.printf.shuttle_tracker.View.Splash;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;

import com.printf.shuttle_tracker.R;
import com.printf.shuttle_tracker.View.Map.MapsActivity;

import java.util.Timer;

public class SplashActivity extends AppCompatActivity {
    public Timer timer;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);
        try {
            timer.wait(1000);
            startActivity(new Intent(this, MapsActivity.class));
        } catch (InterruptedException e) {
            e.printStackTrace();
        } finally {
            startActivity(new Intent(this, MapsActivity.class));
        }
    }

}

File: app/src/main/java/com/printf/shuttle_tracker/View/Splash/SplashActivity.kt
package com.printf.shuttle_tracker.View.Splash

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.content.Intent
import com.printf.shuttle_tracker.R
import com.printf.shuttle_tracker.View.Map.MapsActivity
import java.util.*

class SplashActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)
        try {
            startActivity(Intent(this, MapsActivity::class.java))
        } catch (e: InterruptedException) {
            e.printStackTrace()
        } finally {
            startActivity(Intent(this, MapsActivity::class.java))
        }
    }
}